data/json/tool_qualities.json

```diff
@@ -116,6 +116,11 @@
     "id": "SCREW_FINE",
     "name": "fine screw driving"
   },
+  {
+    "type": "tool_quality",
+    "id": "TWO_WAY_RADIO",
+    "name": "two-way radio"
```

Comment:
I may be missing something, but is this used anywhere?

Reply:
I added the [two-way radio] tool flag. currently the only thing that uses it, is the two-way radio item, but in the future different items could use that flag too.
I thought if you add a tool flag you have to also add it to tool_qualities.json to describe it.

Reply:
There's no need to add a tool quality for two-way radios - we don't have qualities for thermometers, for example, or watches. Describe it in `flags.json`.